data/entities.js

```diff
@@ -7782,6 +7814,7 @@ module.exports = [
     company: 'Rakuten',
     categories: ['ad'],
     domains: ['*.linksynergy.com'],
+    examples: ['tag.amp.rakuten.com'],
```

Comment:
this isn't a valid example for this set of domains, missing `*.rakuten.com`?

Reply:
@patrickhulce  PR had failed when I added *.rakuten.com. Didn't have time to look into it so removed it for now.. Will look at updating Rakuten later.. removing example domain for now.